kinde-j2ee/src/main/java/com/kinde/servlet/KindeLoginServlet.java

```diff
@@ -0,0 +1,44 @@
+package com.kinde.servlet;
+
+import com.kinde.authorization.AuthorizationUrl;
+import com.kinde.token.AccessToken;
+import com.kinde.token.IDToken;
+import com.kinde.token.KindeToken;
+import com.kinde.token.RefreshToken;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.util.List;
+
+@Slf4j
+public class KindeLoginServlet extends HttpServlet {
+
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String code = req.getParameter("code");
+        if (code == null) {
+            // Redirect to the OAuth provider's authorization page
+            AuthorizationUrl authorizationUrl = KindeSingleton.getInstance().getKindeClient().clientSession().authorizationUrl();
+            req.getSession().setAttribute("AuthorizationUrl",authorizationUrl);
```

Comment:
SessionAttributes.AUTHORIZATION_URL constant can be used

Reply:
I cannot find SessionAttributes.AUTHORIZATION_URL but have replaced the AuthorizationUrl with a constant.